test/integration/builder/additional.js

```diff
@@ -986,7 +986,7 @@ module.exports = function(knex) {
         });
     });
 
-    it('Event: does not duplicate listeners on a copy with user params', function() {
+    it('Event: preserves listeners on a copy with user params', function() {
```

Comment:
Probably users are going to be confused if listeners set on original knex suddenly disappear when we copy it with user params, so this is an improvement.